client/state/ui/action-log/reducer.js

```diff
@@ -27,10 +32,15 @@ const relevantTypes = {
 };
 
 const isRelevantAction = ( action ) =>
-	relevantTypes.hasOwnProperty( action.type ) &&
-	( typeof relevantTypes[ action.type ] !== 'function' ||
+	has( relevantTypes, action.type ) && (
+		! isFunction( relevantTypes[ action.type ] ) ||
```

Comment:
None of the currently `relevantTypes` seem to be a function anyway -- they're all just action type (string) constants, aren't they?